### PR TITLE
Garamizo patch 1

### PR DESCRIPTION
**Pull request by garamizo**
Please review this, @casey-collab

### CODE REVIEW

File: english/watched/watched.md [hunk 1]
```diff
@@ -5,3 +5,4 @@ Does the abyss know that you're late with your tax return?
 
 Perhaps you'd like to lighten up and [bake some cookies.](../bake-cookies/cookies.md)
 
+Or evade the tax collection and [settle in Canada](../../Canada/Canada.md).
```
Comment (casey-collab) [inline english/watched/watched.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by garamizo** [comment]
You're right.
It's fixed now, @casey-collab.

**Comment by casey-collab** [comment]
Looks great! :melon:

**Pull request merged**